src/packagedcode/pypi.py

```diff
@@ -71,6 +71,14 @@ class BasePypiHandler(models.DatafileHandler):
     def compute_normalized_license(cls, package):
         return compute_normalized_license(package.declared_license)
 
+    @classmethod
```

Comment:
What do you gain from overriding these two methods with the exact same code that's in the super class?